WorkloadManagementSystem/DB/JobDB.py

```diff
@@ -1513,7 +1513,14 @@ def getSiteMask( self, siteState = 'Active' ):
 
 #############################################################################
   def getSiteMaskStatus( self, sites = None ):
-    """ Get the currently site mask status
+    """ Get the current site mask status
+        :param:sites - A string for a single site to check, or a list
+                       to check multiple sites.
+        :returns:If input was a list, a dictionary of sites, keys are site
+                 names and values are the site statuses. Unknown sites are
+                 not included in the output dictionary.
+                 If input was a string, then a single value with that site's
+                 status, or S_ERROR if the site does not exist in the DB.
     """
     if isinstance(sites, list):
       safeSites = []
```

Comment:
Bug in 3 lines below:
res = self._escapeString( site )  -> ret = self._escapeString( site )